src/plugins/gopts/README.md

```diff
@@ -0,0 +1,34 @@
+- infos = Information about the gopts plugin is in keys below
+- infos/author = Klemens Böswirth <k.boeswirth+git@gmail.com>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = procgetstorage
+- infos/status = recommended productive maintained nodep libc experimental
+- infos/metadata =
+- infos/description = Parses command-line options using elektra-opts
+
+## Introduction
+
+This plugin is very simple in what it does. It uses system specific methods of accessing `argc` and `argv` outside of `main` and then calls
+`elektraGetOpts` to do options processing.
+
+## Usage
+
+The preferred way of using this plugin is via `kdbEnsure`:
+
+```c
+KDB * kdb = kdbOpen (parentKey);
```

Comment:
It would be nice if this code is also as stand-alone minimal-working example available.